### PR TITLE
Giv/pr2 2

### PR DESCRIPTION
**Pull request by givp**
All PR 2.2 updates. See commits for changes.

### CODE REVIEW

**Comment by givp** [comment]
I'm not sure why there this PR is including all previous branches and changes??! But the only changes we care about are "Commits on Feb 9, 2020".

**Comment by nicolasmplus** [comment]
I think master branch does not have those "Merge branch 'vue' into staging" commits.

File: nuxt.config.js [hunk 1]
```diff
@@ -92,10 +92,29 @@ module.exports = {
...
   buildModules: [
     //'@nuxtjs/eslint-module'
-    ['@nuxtjs/google-analytics', {
-      id: process.env.GA_CODE
-    }]
+    '@nuxtjs/google-analytics'
   ],
+
+  features: {
+    transitions: false
```
Comment (nicolasmplus) [inline nuxt.config.js]: What is the use for this? I think this is related to the scroll position being kept when clicking through pages

Comment (givp) [reply nuxt.config.js]: Oh wow, that's unfortunate. It is required to make GA update the page titles correctly for the `collect` request. Setting that to true or removing it results in GA titles not getting updated again 😞 I will try a few different things.

**Pull request merged**